core/modules/system/system.install

```diff
@@ -850,6 +850,33 @@ function system_install() {
   if (valid_email_address($site_mail)) {
     config_set('system.core', 'site_mail', $site_mail);
   }
+
+  // Set the install version to the current version of Backdrop.
+  config_set('system.core', 'install_version', BACKDROP_VERSION);
+
+  // Use install version as initial css update version.
+  $initial_update_version = BACKDROP_VERSION;
+
+  // Process the version to retain major.minor.patch format.
+  if (!empty($initial_update_version)) {
+    // Extract major, minor, and patch versions.
+    preg_match('/^(\d+)\.(\d+)(?:\.(\d+))?/', $initial_update_version, $matches);
+
+    // Build the cleaned-up version.
+    if (!empty($matches)) {
+      $major = $matches[1];
+      $minor = $matches[2];
+      $patch = isset($matches[3]) && is_numeric($matches[3]) ? $matches[3] : '0';
+      $initial_update_version = "$major.$minor.$patch";
+    }
+    else {
+      // Fallback in case of unexpected format.
+      $initial_update_version = '0.0.0';
+    }
+  }
+  config_set('basis.settings', 'install_version_clean', $initial_update_version);
+  config_set('basis.settings', 'css_update_version', $initial_update_version);
+  config_set('basis.settings', 'css_update', 'no_updates');
```

Comment:
I'm not sure any of this should be here. There are a couple problems:

The latest install version doesn't always map to a Basis CSS Update version. For example Backdrop `1.30.1` might not have CSS Updates, but the `css_update_version` will be set to that value, and that's not a valid choice. It would need to select the most recent version that had a CSS update, which would be `1.30.0`.

I would say just put these in the `basis.settings.json` file as hard-coded values. We'll increment them every time we add another CSS update version, since we have to add another option to `basis_supplemental_css_versions()` anyway.

Also, not all sites will install Basis. This probably shouldn't be in `system.install`.